tests/end-to-end/ui_smarti/00-preparation.js

```diff
@@ -0,0 +1,270 @@
+/* eslint-env mocha */
+
+import supertest from 'supertest';
+import {adminUsername, adminPassword} from '../../data/user.js';
+
+export const request = supertest.agent('http://localhost:8080');
+export const rcrequest = supertest.agent('http://localhost:3000');
+export const credentials = {
+	username: 'admin',
+	password: 'admin'
+};
+const clientconfig = '{"queryBuilder":[{"_class":"io.redlink.smarti.model.config.ComponentConfiguration","name":"conversationmlt","displayName":"conversationmlt","type":"conversationmlt","enabled":true,"unbound":false,"pageSize":3,"filter":["support_area"]},{"_class":"io.redlink.smarti.model.config.ComponentConfiguration","name":"conversationsearch","displayName":"conversationsearch","type":"conversationsearch","enabled":true,"unbound":false,"pageSize":3,"filter":["support_area"]}]}';
+
+describe('[Smarti Connection]', () => {
+
+	describe('[Status]', function () {
+		describe('health', () => {
+			it('Smarti should be UP', (done) => {
+				request.get('/system/health')
+					.expect(200)
+					.expect('Content-Type', 'application/vnd.spring-boot.actuator.v1+json;charset=UTF-8')
+					.expect((res) => {
+						expect(res.body).to.have.property('status', 'UP');
+					})
+					.end(done);
+			});
+		});
+		describe('Info', () => {
+			it('Access Smarti info', (done) => {
+				request.get('/system/info')
+					.expect(200)
+					.expect('Content-Type', 'application/vnd.spring-boot.actuator.v1+json;charset=UTF-8')
+					.end(done);
+			});
+		});
+	});
+
+	describe('[Smarti Configuration]', function () {
+		describe('[Client]', () => {
+			var clientid;
+			var token;
+			var authToken;
+			var userId;
+
+			it('check if client already exists', function (done) {
+				request.get('/client')
+					.auth(credentials['username'], credentials['password'])
+					.expect(200)
+					.expect(function (res) {
+						if (typeof res.body[0] !== 'undefined') {
+							clientid = res.body[0].id;
```

Comment:
@ThomasRoehl can you please make the selection of the client more robust? It was good if tests could also run with other clients present in the Smarti-server